src/neynar-api/v2/openapi-farcaster/models/ban-list.ts

```diff
@@ -0,0 +1,58 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Farcaster API V2
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 2.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+// May contain unused imports in some cases
+// @ts-ignore
+import { User } from './user';
+
+/**
+ * 
+ * @export
+ * @interface BanList
+ */
+export interface BanList {
```

Comment:
Thanks for catching blocker and blocked! 

I called the field `banned` because it's following the convention used by the mute and block endpoints. The `object` field says `mute`, `block`, or `ban` and the fields are `muted`, `blocked`, or `banned` and the timestamp.